wiki/Lua-Hyperspace-Module.md

```diff
@@ -1124,6 +1124,244 @@ function player_crew_health_boost()
+- `void CheckForTeleport()`
+- `bool CheckRoomPath(int roomId)`
+- `void CheckSkills()`
+- `void Cleanup()`
+- `void ClearPath()`
+- `void ClearPosition()`
+- `void ClearTask()`
+- `void Clone()`
+- `void CloseDoorBehind(Door *door)`
+- `bool ContainsPoint(int x, int y)`
+- `void CycleColorLayer(int unk)`
+- `bool DirectModifyHealth(float health)`
+- `void EmptySlot()`
+- `Slot FindSlot(int roomId, int slotId, bool closeEnough)`
+- `void ForceMindControl(bool force)`
+- `Point GetFinalGoal()`
+- `int GetIntegerHealth()`
+- `Point GetLocation()`
+- `std::string GetLongName()`
+- `float GetMoveSpeed()`
+- `std::string GetName()`
+- `bool GetNewGoal()`
+- `Point GetNextGoal()`
+- `int GetRepairingId()`
+- `bool GetResisted()`
+- `Slot GetSavedPosition()`
+- `static int __stdcall GetSkillFromSystem(int systemId)`
+- `int GetSkillLevel(int skillId)`
+- `float GetSkillModifier(int skillId)`
+- `std::pair<int, int> GetSkillProgress(int skillId)`
+- ~~`static std::string __stdcall GetSkillTooltip(int skillId, int skillLevel, std::pair<int, int> progress, bool infoScreen)`~~
+- `std::string GetTooltip()`
+- `void IncreaseSkill(int skillId)`
+- `void InitializeSkills()`
+- `bool IsBusy()`
+- `bool IsManningArtillery()`
+- `void Kill(bool noClone)`
+- `void MasterSkill(int skillId)`
+- `void ModifyHealth(float health)`
+- `bool MoveToRoom(int roomId, int slotId, bool forceMove)`
+- `bool NeedFrozenLocation()`
+- `bool NeedsSlot()`
+- `bool OnInit()`
+- `void OnRenderHealth()`
+- `void OnRenderPath()`
+- `void RenderSkillUpAnimation(Point pos)`
+- `bool Repairing()`
+- `bool RepairingFire()`
+- `bool RepairingSystem()`
+- `void Restart()`
+- `bool RestorePosition()`
+- `bool Sabotaging()`
+- `void SavePosition()`
+- `Pointf SelectSabotageTarget()`
+- `void SetCloneReady(bool cloneReady)`
+- `void SetCurrentShip(int shipId)`
+- `void SetCurrentSystem(ShipSystem *sys)`
+- `void SetCurrentTarget(CrewTarget *target, bool unk)`
+- `void SetDamageBoost(float damageBoost)`
+- `void SetDeathNumber(int deathNum)`
+- `void SetFrozen(bool frozen)`
+- `void SetFrozenLocation(bool frozenLocation)`
+- `void SetHealthBoost(int healthBoost)`
+- `void SetMedbay(float health)`
+- `void SetMindControl(bool controlled)`
+- `void SetName(TextString *name, bool force)`
+- `bool SetPath(Path *path)`
+- `void SetPosition(Point pos)`
+- `void SetResisted(bool resisted)`
+- `void SetRoom(int roomId)`
+- `void SetRoomPath(int slotId, int roomId)`
+- `void SetSavePosition(Slot position)`
+- `void SetSex(bool male)`
+- `void SetSkillProgress(int skillId, int skillLevel)`
+- `void SetTask(CrewTask task)`
+- `void StartRepair(Repairable *toRepair)`
+- `void StartTeleport()`
+- `void StartTeleportArrive()`
+- `void StopRepairing()`
+- `void UpdateHealth()`
+- `void UpdateMovement()`
+- `bool WithinRect(int x, int y, int w, int h)`
+- ~~`void constructor(CrewBlueprint &blueprint, int shipId, bool intruder, CrewAnimation *animation)`~~
+- ~~`void destructor()`~~
+### Fields
```

Comment:
Should have a newline between the end of the Methods section and the beginning of the Fields section.